README.md

```diff
@@ -88,72 +88,91 @@ Make sure the following properties are part of your `package.json`.
 In your main package file, add the following methods and replace
 `your-package-name` with your package name.
 
-```coffeescript
-consumeToolBar: (toolBar) ->
-  @toolBar = toolBar 'your-package-name'
+```js
+let toolBar;
 
-  # Add buttons and spacers here...
+export function consumeToolBar(toolBar) {
+  toolBar = toolBar('your-package-name');
```

Comment:
Doesn't the argument `toolBar` and the global `toolBar` variable conflict with each other?